### ISSUE TITLE
`makedns -n` will discard all the customized parts in /etc/named.conf file  

### ISSUE THREAD

**Issue by immarvin** [xcat2/xcat-core#5897]
Currently, `makedns -n` will regenerated the /etc/named.conf file and discard all customized parts made by Admin. We should enhance the `makedns -n` to restrict the configuration xCAT relies on

**Comment by kcgthb** [comment]
Yes! That would be awesome. I reported this in #2423 about 2 years ago, and proposed a minimal change to be able to include an external config file in #4497.

But I agree that a better solution would be for xCAT to only generate a portion of `named.conf`, so local customizations could be kept intact when doing `makedns -n`. 
Maybe by having the xCAT-generated portion an included file itself? Like having `makedns` add a `include /etc/named/xcat.conf` line in `named.conf` and generate all the xCAT config in that `/etc/named/xcat.conf` file instead of `named.conf`?

Also,the exact same principle should also apply to `makehosts`, `makentp` and `makedhcp`.
